update requirements.txt and setup.py for python 3.9

The command `pip install -r requirements.txt` works for virtualenv with version 3.5. It does not with 3.9. python 3.5 is also going to be deprecated. See the warning `DEPRECATION: Python 3.5 reached the end of its life on September 13th, 2020. Please upgrade your Python as Python 3.5 is no longer maintained. pip 21.0 will drop support for Python 3.5 in January 2021. pip 21.0 will remove support for this functionality.`.

I have put the logs of the errors.

```
gcc -pthread -Wno-unused-result -Wsign-compare -DNDEBUG -g -fwrapv -O3 -Wall -fPIC -DCYTHON_CLINE_IN_TRACEBACK=0 -I/usr/include/libxml2 -Isrc -Isrc/lxml/includes -I/usr/local/include/python3.9 -c src/lxml/etree.c -o build/temp.linux-x86_64-3.9/src/lxml/etree.o -w
  src/lxml/etree.c: In function ‘__Pyx_modinit_type_init_code’:
  src/lxml/etree.c:231155:32: error: ‘PyTypeObject {aka struct _typeobject}’ has no member named ‘tp_print’; did you mean ‘tp_dict’?
     __pyx_type_4lxml_5etree_Error.tp_print = 0;
                                  ^
  src/lxml/etree.c:231163:36: error: ‘PyTypeObject {aka struct _typeobject}’ has no member named ‘tp_print’; did you mean ‘tp_dict’?
     __pyx_type_4lxml_5etree_LxmlError.tp_print = 0;
```


For the moment, if you met a similar issue, you can skip with `virtualenv --python=/usr/bin/python3.5 venv-3-5` instead of `virtualenv venv -p python3` on the build instruction of the exodus core readme.md.

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -15,12 +15,17 @@ on:
 jobs:
   build:
     runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+        python-version:
+          - 3.7
+          - 3.9
     steps:
       - uses: actions/checkout@v2
-      - name: Set up Python 3.7
+      - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
-          python-version: 3.7
+          python-version: ${{ matrix.python-version }}
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip

--- a/exodus_core/analysis/static_analysis.py
+++ b/exodus_core/analysis/static_analysis.py
@@ -1,24 +1,22 @@
-from collections import namedtuple
-from cryptography.x509.name import _SENTINEL, ObjectIdentifier, _NAMEOID_DEFAULT_TYPE, _ASN1Type, NameAttribute
-from hashlib import sha256, sha1
-from PIL import Image
-from tempfile import NamedTemporaryFile, TemporaryDirectory
 import binascii
-import dhash
 import itertools
 import logging
 import os
 import re
+import subprocess
+import zipfile
+from collections import namedtuple
+from future.moves import sys
+from hashlib import sha256, sha1
+from tempfile import NamedTemporaryFile, TemporaryDirectory
+
+import dhash
 import requests
 import six
-import subprocess
-# import time
-import zipfile
-
 from androguard.core.bytecodes import axml
 from androguard.core.bytecodes.apk import APK
-from future.moves import sys
-# from gplaycli import gplaycli
+from cryptography.x509.name import _SENTINEL, ObjectIdentifier, _NAMEOID_DEFAULT_TYPE, _ASN1Type, NameAttribute
+from PIL import Image
 
 PHASH_SIZE = 8
 

--- a/exodus_core/analysis/test_exodus_analyze.py
+++ b/exodus_core/analysis/test_exodus_analyze.py
@@ -73,8 +73,8 @@ class TestExodus(unittest.TestCase):
         diff_1 = sa.get_icon_similarity(phash_1, phash_2)
         diff_2 = sa.get_icon_similarity(phash_2, phash_1)
 
-        self.assertEqual(diff_1, 0.7265625)
-        self.assertEqual(diff_2, 0.7265625)
+        self.assertEqual(diff_1, 0.734375)
+        self.assertEqual(diff_2, 0.734375)
 
     def test_icon_similarity_different_similarities(self):
         sa = StaticAnalysis()

--- a/requirements.txt
+++ b/requirements.txt
@@ -2,7 +2,7 @@ androguard==3.3.5
 cryptography==3.3.2
 dhash==1.3
 jellyfish==0.5.6
-Pillow==6.2.2
+Pillow==8.3.2
 requests==2.26.0
 six==1.15.0
 traitlets==4.3.2

--- a/setup.py
+++ b/setup.py
@@ -41,7 +41,7 @@ install_requires = [
     'cryptography==3.3.2',
     'dhash==1.3',
     'jellyfish==0.5.6',
-    'Pillow==6.2.2',
+    'Pillow==8.3.2',
     'requests==2.26.0',
     'six==1.15.0',
     'traitlets==4.3.2'

